app/service/form_service.rb

```diff
@@ -11,4 +11,23 @@ def path_for_state
 
     form_path(@form.id)
   end
+
+  def add_to_default_group!(current_user)
+    return if current_user.trial?
+
+    org = current_user.organisation
+
+    if org.default_group.nil?
```

Comment:
This could probably be in a separate method

Reply:
Yeah, this feels sensible to me too

Reply:
I think this is a really good point and refactoring this code would make sense if we were planning on keeping it around beyond the migration.

As we've tested it along with our rake task we'd prefer to keep it as is for now, if thats ok.

Reply:
Worth noting that we did try having it in a different method when writing the code originally, but we found that it didn't bring much benefit.